src/Stache/Query/Builder.php

```diff
@@ -131,14 +131,55 @@ protected function filterWhereBasic($values, $where)
     protected function filterWhereIn($values, $where)
     {
         return $values->filter(function ($value) use ($where) {
+            if (is_array($value)) {
+                return count(array_intersect($value, $where['values']));
+            }
+
             return in_array($value, $where['values']);
         });
     }
 
     protected function filterWhereNotIn($values, $where)
     {
         return $values->filter(function ($value) use ($where) {
+            if (is_array($value)) {
```

Comment:
Remove this?

Reply:
Yep my bad, forgot to remove the previous approach